fluentd/filter_concat.rb

```diff
@@ -0,0 +1,254 @@
+module Fluent
```

Comment:
We should move this into the `lib` dir so that we can add tests if needed based on any changes we need.  I assume this is otherwise a straight copy